testsuite/MDAnalysisTests/__init__.py

```diff
@@ -98,7 +98,11 @@
 logger = logging.getLogger("MDAnalysisTests.__init__")
 
 # keep in sync with RELEASE in setup.py
+<<<<<<< HEAD
 __version__ = "1.0.0"
+=======
+__version__ = "2.0.0-dev0"
+>>>>>>> upstream/develop
```

Comment:
This is a merge gone wrong... not sure where this happened but needs to be fixed.

Reply:
Sorry I made an absolute mess. I'll clean it up.